src/Microsoft.FeatureManagement/FeatureManagementOptions.cs

```diff
@@ -22,5 +22,12 @@ public class FeatureManagementOptions
         /// The default value is true.
         /// </summary>
         public bool IgnoreMissingFeatures { get; set; } = true;
+
+        /// <summary>
+        /// Controls the behavior of feature definition provider when "FeatureManagement" section in the configuration is missing.
+        /// If missing configuration section is not ignored, the feature definition provider will fall back to the root of the provided configuration.
+        /// The default value is false.
+        /// </summary>
+        public bool IgnoreMissingConfigurationSection { get; set; }
```

Comment:
I think the naming here could be a little clearer. We're not ignoring, rather we're no longer returning the top level configuration children.

Perhaps something like: "DefaultToTopLevelConfiguration" with a default of true?

Reply:
Or `RequireFeatureManagementSection`

Reply:
I vote for RequireFeatureManagementSection.